### ISSUE TITLE
accessibility issues with EU Captcha

### ISSUE THREAD

**Issue by marius-salomeia** [pwc-technology-be/EU-CAPTCHA#176]
We are integrating EU Capthca in contact form and we noticed following accessibility issues:
**Captcha text:** Capitalized checkbox is not controllable by keyboard. Moreover, its label is not translated, it remains in English regardless of the language selected.
**The color** of the Help icon (bulb) is low in contrast with 1.97:1. To do well, the contrast ratio with the background should be greater than or equal to 3:1.
**Captcha image:** nothing special except that it is obviously not intended for visually impaired people.

**Comment by YvesMeersmans** [comment]
@Eddy-Hoevenaers Can you please look into this one? We were recently in touch with Marius as well. 
@marius-salomeia Can you specify in the meantime whether you would like to implement the EU CAPTCHA by yourself, and if you wish to share, in which application? Was this related to the recent communications? Sorry for replying with a serious delay.

**Comment by Eddy-Hoevenaers** [comment]
Hi Marius,

In the latest update i added all the translations, also for the capitalized checkbox.
For the visually impaired people we added the spoken option.

I will look into the Help icon contrast and the checkbox controlled by keybord issue

**Comment by Eddy-Hoevenaers** [comment]
With the latest update you can control the checkbox with use of the 'TAB' key and select/deselct it with the space key

**Comment by mariussalomeia** [comment]
Hello, 
yes the improvements are linked to the recent communication. 
In parallel we are trying to integrate the CAPTCHA into OP Portal. We did a PoC and we are evaluating now before rolling out to all features.
